plugins/backend/__init__.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
```

Comment:
The original class backend, that renders the actual backend app, gets separated into a class in BackendCore.py 
A similar class BackendBlockly.py provides the new addon for the Blockly Logics Editor. (Maybe a concept for adding functionality to backend?)